src/components/gridlines.ts

```diff
@@ -42,6 +64,36 @@ export class Gridlines extends Component {
     }
   }
 
+  public betweenX(_betweenX: boolean): this;
```

Comment:
`minor` isn't quite what he's going for i think. `minor` implies there are two sets: `major` and `minor`. but with `between`, that means take the normal grid lines and move them elsewhere